### PR TITLE
Add Spektrum Bar to Berlin bars

### PR DESCRIPTION
**Pull request by mamhoff**
This is a great place, and it has been missing sorely from this repository before.

This commit adds the Spektrum bar to the places accessible to the underground.

### CODE REVIEW

File: Berlin/bars/spektrum.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# kaufbar
+
```
Comment (starkcoffee) [inline Berlin/bars/spektrum.md]: Hmm the name is not matching?

**Comment by starkcoffee** [comment]
Hey this looks really cool! Just noticed a small problem with the name, please update.

File: Berlin/bars/spektrum.md (hunk 1 above)
Comment (mamhoff) [inline Berlin/bars/spektrum.md]: Oops! That must've been a copypasta error. 
Let me just quickly amend this commit.

**Comment by mamhoff** [comment]
Updated the PR with correct name. Thanks for the meticulous eye.

**Comment by starkcoffee** [comment]
Specktrum-tacular! 🍻

**Pull request merged**